exercises/ex_atm.py

```diff
@@ -0,0 +1,162 @@
+"""ATM Exercise"""
+
+import sys
+
+FILE_PATH_INDEX = 1  # describe placement of the argument containing file path.
+PASSWORD_INDEX = 0
+BALANCE_INDEX = 1
+END_ATM_SERVICE = "-1"
+
+
+def store_file_content(file_path):
+    """Read file and move atm information to dictionary.
+    The lines of the file will contain user's id, balance and password,
+    with comma's separating them.
+
+    Args:
+        file_path (str): path to atm information file.
+
+    Return:
+        dict. with user's id as keys,
+        and user's password and balance in a list as values.
```

Comment:
if you have more then 1 line for return \ arg doc, the lines after the first one should start with indent
for example:
dict. something
____continue something (it doesn't show space in comment, so I putted _ instead)